data/json/items/tool/misc.json

```
@@ -707,30 +707,56 @@
     "id": "teleumbrella",
     "type": "ARMOR",
     "category": "tools",
-    "name": { "str": "telescoping umbrella" },
-    "description": "A telescoping umbrella which collapses down for easy storage, useful for keeping dry when wielded.",
-    "weight": "118 g",
-    "volume": "500 ml",
+    "name": { "str": "telescoping umbrella (collapsed)", "str_pl": "telescoping umbrellas (collapsed)" },
+    "description": "A telescoping umbrella which has been collapsed down for easy storage.  May be used to keep dry when extended and wielded.",
+    "weight": "411 g",
+    "volume": "1500 ml",
     "price": 12000,
     "price_postapoc": 100,
-    "to_hit": 1,
-    "bashing": 1,
+    "to_hit": -2,
+    "bashing": 3,
     "material": [ "plastic", "aluminum" ],
     "symbol": ",",
     "color": "dark_gray",
     "sided": true,
     "warmth": 0,
     "material_thickness": 1,
     "techniques": [ "WBLOCK_1" ],
-    "flags": [ "ALLOWS_NATURAL_ATTACKS", "BELT_CLIP", "BELTED", "FRAGILE", "OVERSIZE", "RAIN_PROTECT" ],
+    "flags": [ "ALLOWS_NATURAL_ATTACKS", "BELT_CLIP", "BELTED", "FRAGILE", "OVERSIZE" ],
     "armor": [
       {
         "encumbrance": 25,
         "coverage": 50,
         "covers": [ "hand_l", "hand_r" ],
-        "specifically_covers": [ "hand_wrist_l", "hand_wrist_r" ]
+        "specifically_covers": [ "hand_wrist_l", "hand_wrist_r" ],
+        "material": [ { "type": "plastic", "covered_by_mat": 100, "thickness": 1 } ]
       }
-    ]
+    ],
+    "use_action": { "menu_text": "Expand", "type": "transform", "target": "teleumbrella-extended", "msg": "You open your umbrella." }
```

For consistency with name and description of extended form.
```suggestion
    "use_action": { "menu_text": "Extend", "type": "transform", "target": "teleumbrella-extended", "msg": "You open your umbrella." }
```